inferentia/scripts/setup.sh

```diff
@@ -133,13 +133,16 @@ make triton-python-backend-stub -j16
 pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
 conda config --env --add channels https://conda.repos.neuron.amazonaws.com
 
-if [ $USE_TENSORFLOW -eq 1 ]
-then
+if [ $USE_TENSORFLOW -eq 1 ] || [ $USE_TENSORFLOW -eq 2 ]; then
     conda install tensorflow-neuron pillow -y
```

Comment:
Don't we need to install the specific version of tensorflow-neuron here as well?

Reply:
No I think the --upgrade step will deal with the resolution